client/my-sites/themes/theme-showcase.jsx

```diff
@@ -204,13 +204,16 @@ class ThemeShowcase extends React.Component {
 				.sort( ( a, b ) => a.order - b.order )
 		);
 
+		const showBanners = currentThemeId || ! siteId || ! isLoggedIn;
+
 		const themeBanners = omit(
+			// We don't want to advertise the theme that's already active.
```

Comment:
Nitpick - it's a bit nasty to have the comment _inside_ the function call. Maybe move it up one line?

Reply:
Ha! That comes from Prettier, moving...